README.md

```diff
@@ -59,7 +59,7 @@ Required options:
 
 Additional options:
   --constructorBytecode <CONSTRUCTOR_BYTECODE>  0x-prefixed ABI encoded byte string representing the constructor arguments. Required if the constructor has arguments.
-  --licenseType <LICENSE>         License type for the contract. Recommended if verifying source code. Defaults to "None".
+  --licenseType <LICENSE>         License type to display on block explorers for verified source code. See https://etherscan.io/contract-license-types for supported values and use the string found in brackets, e.g. MIT
```

Comment:
What about those values with a space (e.g. GNU GPLv2), will they keep working?

Reply:
Changed the examples to use quotes, which will handle spaces.